python/paddle/fluid/contrib/tests/README.md

```diff
@@ -0,0 +1,62 @@
+PaddlePaddle supports offline INT8 calibration to accelerate the inferene speed. In this document, we provide the instructions on how to enable INT8 calibration and show the results in both accuracy and performance.
+
+## 0. Prerequisite
+You need to build PaddlePaddle python package (e.g., paddlepaddle-0.0.0-cp27-cp27mu-linux_x86_64.whl) and install the package successfully.
+
+## 1. How to generate INT8 model
+You can refer to the unit test in test_calibration.py. Basically, there are three steps:
+* Construct calibration object. The required parameters are: 1) **program**: FP32 program; 2) **pretrained_model**: FP32 pretrained model; 3) **algo**: calibration algorithm; default is np.max, the alternative is KL (Kullback–Leibler divergence); 4) **exe**: executor; 5) **output**: INT8 model; 6) **feed_var_names**: feed dict; and 7) **fetch_list**: fetch targets. Here is the example:
+
+            calibrator = int8_utility.Calibrator( # Step 1
+                program=infer_program,
+                pretrained_model=model_path,
+                algo=algo,
+                exe=exe,
+                output=int8_model,
+                feed_var_names=feed_dict,
+                fetch_list=fetch_targets)
+
+* Sample data. Call the calibrator.sample_data() after executor run.
+            _, acc1, _ = exe.run(
+                program,
+                feed={feed_dict[0]: image,
+                      feed_dict[1]: label},
+                fetch_list=fetch_targets)
+
+                calibrator.sample_data() # Step 2
+
+* Save INT8 model. Call the calibrator.save_int8_model() after sampling over specified iterations (e.g., iterations = 50)
+            calibrator.save_int8_model() # Step 3
+
+Please note that we assume FP32 pre-trained model has the `__model__ ` file with image and label feed variables which will be used by the accuracy measurememnt.
```

Comment:
please make line31 simple to read.

Reply:
Done. Moved to the notes in Section Results.